### ISSUE TITLE
[Suggestion] Flat decorative bricks

### ISSUE THREAD

**Issue by PopkaAkula** [tsunamayo/Starship-EVO#1928]
### Short Description of the suggestion.

![image](https://user-images.githubusercontent.com/37738021/80867619-5f977a80-8c95-11ea-8389-d5fe6291160c.png)

As of now the current roster of decorative bricks is pretty limited, they are all sloped bricks, and it would be nice to have something like these decorative bricks but flat.

**Comment by schlid** [comment]
yes i agree there were some blocks in the old codex that added a little something something. Granted the bump mapped stickers are quite useful but adding something like this suggestion would give us far more options
